CNAME

```diff
@@ -1 +0,0 @@
-get.composable.finance
```

Comment:
it was github pages named with dns afaik and we had only script there.  may be should clean up DNS record @haroldsphinx

Reply:
and clean up github side setting for github pages too cc @KaiserKarel

Reply:
I think get.composable.finance was from a legacy script way, way back by Oleksii, so should be easy to remove now.
What do we need to remove pages side?